atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -95,11 +108,48 @@ def push_with_skopeo(self, registry_image, insecure, docker_push_secret):
             self.log.error("push failed with output:\n%s", e.output)
             raise
 
+    def source_get_unique_image(self):
```

Comment:
It feels like tag_and_push should just apply the tags already decided on for this image. Maybe The source container build plugin should set tag_conf?

Reply:
I don't know, then again source container build should just build it :)
I'd rather keep logic here

Reply:
Can we run the `tag_from_config` plugin? That ought to set tag_conf I think.

May need a change in `render_tag_from_config` to make sure we only put the unique tag in its config.

Reply:
Seems like the team preference is to figure out the unique tag here rather than in osbs-client.